lib/game.rb

```diff
@@ -1,41 +1,51 @@
 require_relative "display"
 require_relative "messages"
-require_relative "rules"
 require_relative "player"
+require_relative "human_player"
+require_relative "computer_player"
+require_relative "rules"
+require_relative "move_validator"
+require_relative "set_up"
 
 class Game
 
-  def initialize(player_one, player_two, display, messages)
-    @player_one = player_one
-    @player_two = player_two
+  def initialize(display, rules, set_up)
     @display = display
-    @messages = messages
+    @rules = rules
+    @set_up = set_up
   end
 
-  def run
-    welcome_users
-    @player_one.choose
-    @player_two.choose
-    determine_winner
-    present_winner
+  def play
+    @display.welcome_users
+    @display.choose_player
```

Comment:
Similarly to comment below, choosing which player goes first is part of the game setup